### PR TITLE
feat(firestore): add limit_to_last samples

### LINKED ISSUE #4195: Firestore: add samples for limit_to_last feature
Add samples for `limit_to_last` feature implemented with [this PR](https://github.com/googleapis/python-firestore/pull/57/files)

### PR DESCRIPTION
**Pull request by IlyaFaer**
Closes #4195

### CODE REVIEW

**Comment by IlyaFaer** [comment]
@BenWhitehead, @crwilcox, I've added the samples. Taken the tag name from the Java's samples

**Comment by BenWhitehead** [comment]
@IlyaFaer Looks like all three builds are failing with the following traceback

```
Traceback (most recent call last):
  File "/workspace/firestore/cloud-client/snippets_test.py", line 239, in test_order_limit_to_last
    snippets.order_limit_to_last()
  File "/workspace/firestore/cloud-client/snippets.py", line 579, in order_limit_to_last
    query = cities_ref.order_by("name").limit_to_last(2)
AttributeError: 'Query' object has no attribute 'limit_to_last'
```

**Comment by IlyaFaer** [comment]
@BenWhitehead, yeah, I suppose, it'll be failing until the next Firestore release. Locally I've replaced the Python original Firestore package with the last repository version, and it's working fine:

Безымянный

File: firestore/cloud-client/snippets_test.py [hunk 1]
```diff
@@ -235,6 +235,10 @@ def test_order_where_limit():
     snippets.order_where_limit()
 
 
+def test_order_limit_to_last():
+    snippets.order_limit_to_last()
```
Comment (tmatsuo) [inline firestore/cloud-client/snippets_test.py:239]: Does it make sense to check the output?

**Comment by busunkim96** [comment]
If you want to check that CI passes, you can temporarily change the `requirements.txt` to install GitHub from the master branch.

https://github.com/q-logic/python-docs-samples/blob/limit_to_last_samples/firestore/cloud-client/requirements.txt#L1

```
git+https://github.com/googleapis/python-firestore.git
```

**Comment by IlyaFaer** [comment]
@busunkim96, sounds interesting, thanks!
@tmatsuo, could you run kokoro checks? I don't have rights to add labels 😞 
I've tried to change `requirements.txt` as adviced, let's give it a try

**Comment by BenWhitehead** [comment]
Kokoro builds triggered

**Comment by IlyaFaer** [comment]
@BenWhitehead, thanks! Checks are green. Please, acknowledge, and I'll revert `requirements.txt`
I've pinged the question of adding me into GoogleCloudPlatform, so, hopefully, I'll get some abilities soon

**Comment by IlyaFaer** [comment]
New Firestore client released, I've reverted `requirements.txt` changes, and now it's working fine.
Dropping "do not merge" label

File: firestore/cloud-client/snippets.py [hunk 2]
```diff
@@ -572,6 +572,16 @@ def order_where_limit():
...
 
 
+def order_limit_to_last():
+    db = firestore.Client()
+    # [START fs_order_by_name_limit_query]
+    cities_ref = db.collection("cities")
+    query = cities_ref.order_by("name").limit_to_last(2)
+    results = query.get()
+    # [END fs_order_by_name_limit_query]
+    print(results)
```
Comment (busunkim96) [inline firestore/cloud-client/snippets.py:582]: Should the print be placed within the region tags?

File: firestore/cloud-client/snippets_test.py (hunk 1 above)
Comment (busunkim96) [reply firestore/cloud-client/snippets_test.py:239]: @IlyaFaer Could you comment on this?

You can return the results in the function and check them here if that's easier than checking the string output.

```py
results = snippets.order_limit_to_last()
assert ...
```

File: firestore/cloud-client/snippets.py (hunk 2 above)
Comment (IlyaFaer) [reply firestore/cloud-client/snippets.py:582]: In other Firestore snippets prints are out of region tags, so it looks like no

File: firestore/cloud-client/snippets_test.py (hunk 1 above)
Comment (IlyaFaer) [reply firestore/cloud-client/snippets_test.py:239]: Oh, that slipped my radars. Yeah, I thought about that, but most of the snippets test are written this way - just a call to snippet function. So, as I see it, if we're gonna add an `assert`, we should do this for all the tests to keep the fashion, which probably should be done in a separate issue. Adding an `assert` into this single test, while other similar tests doesn't have it, looks weird. WDYT?

**Comment by IlyaFaer** [comment]
Friendly ping

File: firestore/cloud-client/snippets_test.py (hunk 1 above)
Comment (tmatsuo) [reply firestore/cloud-client/snippets_test.py:239]: Alright.

Comment (tmatsuo) [reply firestore/cloud-client/snippets_test.py:239]: Can you create an issue for it?

Comment (IlyaFaer) [reply firestore/cloud-client/snippets_test.py:239]: Sure, will make a /cc to you when created. And I think I'll do this

**Pull request merged**